mkdocs.yml

```diff
@@ -2,5 +2,28 @@ site_name: Chatterino
 theme:
   name: material
   logo: https://chatterino.com/img/icon-256.png
+  favicon: https://chatterino.com/img/icon-256.png
+  palette:
+    scheme: preference
+  features:
+    - instant
+  icon:
+    repo: fontawesome/brands/github
 plugins:
   - search
+  - git-revision-date-localized:
+      type: datetime
+      fallback_to_build_date: true
+repo_url: https://github.com/chatterino/chatterino2
```

Comment:
This linking to the chatterino repo instead of the wiki seems unintuitive

Reply:
Entirely up to you. The wiki is about chatterino2, so I thought it'd be most appropriate to link to the chatterino2 repo. FYR I also have this linked in the social icons at the bottom of the page.

Reply:
And I've removed the edit icons (they didn't look good to me), I can have them re-added if you want